xhydro/optimal_interpolation/optimal_interpolation_fun.py

```diff
@@ -451,6 +465,10 @@ def execute_interpolation(
         centroid_lon_obs=filtered_dataset["centroid_lon"].values,
```

Comment:
GitHub ne me donne pas accès, mais il faudrait mettre la même chose ici, avec `if p1_bnds is None: (...)`